### ISSUE TITLE
Affiliation constraints on TAG membership

### ISSUE THREAD

**Issue by martinthomson** [w3c/w3process#817]
In #809, something of a side discussion has started about the affiliation constraints in process for TAG members.  I have no particular opinion on this at this stage, but I do have the ability to open issues.

It was observed that what was a hard rule has been softened somewhat over time, but some people expressed an interest in maybe loosening those constraints further.

I've read that conversation and am moved to ask: how much does the following text from [Section 3.3.3.1](https://www.w3.org/Consortium/Process/Drafts/#elected-expectations) hold water?

> Individuals elected or appointed to the Advisory Board or TAG act in their personal capacity, to serve the needs of the W3C membership as a whole, and the Web community. Whether they are Member representatives or Invited Experts, their activities in those roles are separate and distinct from their activities on the Advisory Board or TAG.

Having come from the IETF, we all know that there is some amount of fiction behind the idea that someone acts solely in their personal capacity.  As is often said, the ability of a person to ignore the truth is proportional to the extent to which their paycheck depends on doing so.  That said, the sorts of situation where these process safeguards apply are rarely simple disputes about fact.

From there, to what extent does the process need to safeguard against that possibility?

cc: @dwsinger, @frivoal, @chrisn, @cwilso, @LeaVerou, @mnot.

**Comment by frivoal** [comment]
I think that in practice, as far as I have been able to observe, TAG and AB members (as well as Board members, for that matter) have been good at not putting the interests of their employer ahead of the collective interest (even in some cases where it was clear that the interest of their employer went the other way).

At the same time, that does not mean that their general perspective about things isn't colored by their usual work environment, colleagues, and areas of focus. I think that's absolutely normal and healthy, as long as not everybody comes from the same perspective.

Also, we should not only strive for minimizing the reality of conflicts of interests, but also the appearance of them. Even when everybody behaves admirably, if external observers aren't convinced that's the case, a suspicion of impropriety can affect the ability to conduct work in an appeased manner.

Overall, I am not convinced that the difference of treatment between AB and TAG in https://www.w3.org/Consortium/Process/Drafts/#AB-TAG-constraints is particularly useful, and wouldn't mind unifying the two, but I'd say that aside from that, the Process seems about right to me.

**Comment by mnot** [comment]
My concern is largely whether the constraints are proportional to the risks they're trying to prevent, and their impacts on the ability of individuals to serve as well as the availability of their services to the community.

Clearly, 10 or even six out of 12 people on the TAG shouldn't have the same affiliation; that's easy to connect to the principle of openness and assuring that we aren't captured. I'd assert that two out of 12 doesn't have anything like the same risk profile -- they cannot unilaterally make decisions on behalf of the TAG, and there is still significant diversity within the body.

I'm less concerned about the impact of the constraint upon the AB, but I think that it could also be relaxed to two for symmetry if folks like that.

**Comment by dwsinger** [comment]
I have not experienced it at W3C, but in other bodies I have strongly suspected that people were not speaking their own mind, but were influenced by their employer's opinion. Yes, the process says that they act in their personal capacity, but which has the greater strength – process or paycheck?

We also want diverse _employers_ on these bodies, to give the greatest opportunity for diverse perspective.

I don't think it's broken, there is a real risk behind changing it – and we should thus leave it alone.

**Comment by cwilso** [comment]
As there have been several affiliation changes announced shortly after elections, I worry that allowing multiple seat-holders for longer than we currently do will encourage this behavior in the future.  I think the TAG rule currently in the process ("don't kick anyone out due to an affiliation change, but ensure there are not multiple seat-holders with the same affiliation at the next regular election") is a fairly even solution.  I would agree with @frivoal that adopting that for the AB would be fine, too.

(I'll note for the record we have no affiliation-change semantics on the Board of Directors, which I raised as an issue: https://github.com/w3c/AB-memberonly/issues/175 and https://github.com/w3c/board/issues/18.). That would of course be up to the Bylaws, not the Process.

**Comment by mnot** [comment]
Regarding Directors, see Article IV Section 2. Qualification is not just for election, it's continuing; see references to "disqualification".

**Comment by dwsinger** [comment]
So, we have dual-affiliation semantics for the Board, but we do not have a "seat is vacated at teh next election by a change" rule to match "If a participant changes affiliation, but the participation constraints are met, that participant’s seat becomes vacant at the next regularly scheduled election for that group." [rule](https://www.w3.org/2023/Process-20231103/#AB-TAG-vacated),

**Comment by cwilso** [comment]
Indeed, that was the first issue I was referring to.  Membership can't respond to an affiliation change for two years in a Director, though it's a year at most for the TAG or AB.

Also, it's not entirely clear in the Bylaws what happens if the constraint is NOT met (for Member-elected Directors).  It would appear from Section 4 that their seats become open for the rest of their term, and an election of some form should be held - Section 4 of Article IV seems to say so ("A vacancy may be filled for the unexpired portion of the term by the class of members with the power to elect or designate the applicable directorship") but gives no detail on how to do that, and Section 3(a) implies that Directors can only be elected at a Meeting ("Seven (7) directors shall be elected from time to time ...by voting of the Consortium Members present in person or represented by proxy at the meeting").

Oddly, also, if an appointed (Director-Elected) Director resigns (or their seat otherwise becomes vacant), it would appear given the current text that any Director-election (appointment) to that seat would be for the remainder of the original term, not an new 2-year term.  That seems... unnecessary.

**Comment by mnot** [comment]
@cwilso I believe that an election is required by the language in the bylaws if a Consortium Director is disqualified. I have a question out to Counsel to confirm that, and to see if we can clarify it in a bylaws update.

**Comment by cwilso** [comment]
@mnot Right, I could see that interpretation of the bylaws, but it also seems like the only opportunity to hold an election (of the Member-elected Directors) is at the annual meeting.

**Comment by mnot** [comment]
@cwilso why do you say that?

**Comment by dwsinger** [comment]
"A vacancy may be filled for the unexpired portion of the term by the class of members with the power to elect or designate the applicable directorship". That means an election for the Consortium Directors. There is specidic provision for meetings other than Annual "Special meetings of any class of members or of all the members may be called by the Board of Directors upon the Requisite Board Vote (as defined in Article IV, Section 10 below). In addition, any twenty (20) Consortium Members may call a meeting of the Consortium Members, any two (2) Partner Members may call a meeting of the Partner Members, and any two (2) Director Members may call a meeting of the Director Members. Business transacted at any special meeting shall be limited to the purpose stated in the applicable notice." One assumes that the purpose of a special meeting would, in this case, be for electing to a vacant seat to fill the unexpired term.

**Comment by LeaVerou** [comment]
TAG Member here, *not* working at a big company. 🙂 

It’s _incredibly_ hard to find people that satisfy all of the following:
1. They have the right expertise (breadth and depth)
2. They have the time & willingness to contribute their expertise 
3. They have the financial support and ability to travel

Statistically, a lot of these people tend to disproportionally congregate in certain large companies that also happen to be major stakeholders of the web, for a number of reasons. So in practice while well-intentioned, the limit being so rigid ends up depriving the TAG from a lot of talent and even a lot of diversity.

There have been [numerous cases](https://tag.w3.org/history/) in TAG’s history that we lost extremely qualified folks due to an affiliation change, and given how small the candidate pool is in the first place, every single case of that happening is incredibly unfortunate. 

I’m not suggesting it should be a free-for-all, but I would love to explore creative ways to relax these limits while preserving the original intent of the rule. Could we perhaps brainstorm some ideas?

**Comment by chaals** [comment]
TL;DR: I don't see a serious problem that justifies the effort necessary to craft something that's actually a net improvement, given that I do see a real risk being addressed more or less adequately by the current approach. I believe this puts me most in alignment with @frivoal's [comment](https://github.com/w3c/w3process/issues/817#issuecomment-1905467036). 

For more detail...

@dwsinger [said](https://github.com/w3c/w3process/issues/817#issuecomment-1906726180)

> I have not experienced it at W3C, but in other bodies I have strongly suspected that people were not speaking their own mind, but were influenced by their employer's opinion. Yes, the process says that they act in their personal capacity, but which has the greater strength – process or paycheck?

I don't see a risk that people will "corruptly" ignore the process for the sake of their paycheck in the community we work in and draw TAG (and AB and Board) members from. But what I see time and again is that companies employ people who disagree vociferously on specific details, but whose own perspectives reflect the core motivations of the company. Thus, they will reliably argue from an entirely personal perspective the most important "talking points" that the company would provide if their role were that of a corporate representative.

The risk here is that a few companies that run in similar directions each have a couple of representatives who have worked together for a long time and share a lot of perspectives in general, leading to a strong orthodoxy that is actually a pretty minoritarian view. Effectively @mnot's "[six of twelve](https://github.com/w3c/w3process/issues/817#issuecomment-1905495739)" is surprisingly easy to walk into, and I think that's a genuinely problematic issue.

It seems easy (for me) to imagine a TAG with one person with substantial expertise in security, one in accessibility, one in privacy, none in regulatory frameworks and law, none in how anti-competitive behaviours destroy markets, none in i18n and localisation, and six who agree between them that while all those things are nice to understand the most important thing for the health of the web is to keep their own sector unregulated in ways that unintentionally produce a very strong bias toward market incumbents. That risks (IMHO) getting a lot of hand-wringing about how hard it is to manage the practical implementation of technologies that support our goals and no actual progress toward those goals because in each instance the TAG comes down against taking the action that would make that happen. All while the individuals refuse to represent anything but their own best intentions and opinions on how to ensure the Web works as well as possible.

On another aspect, @mnot and @LeaVerou once again raised the oft-repeated issue that it's hard to find people good enough to be on the TAG and prepared to do the work. I've heard and considered this argument repeatedly over the last decade or so. I actually find it gets less compelling. I am most concerned that the part I find strongest - people prepared to do the substantial amount of volunteer work expected (unless your salary includes that work in your job description) are hard to find - makes me more concerned that such people will be most "turned off" by being marginalised in a TAG that largely thinks along similar lines on the issues that might make a difference.

**Comment by cwilso** [comment]
@dwsinger hmm, I see.  So you need twenty Members to decide to call a special meeting, at which you could hold a special election.

To respond to the broader thread - I don't particularly think the current rules for TAG affiliation are overly onerous; it attempts to not have multiple TAG seats held by people with a shared affiliation, but to @LeaVerou 's point, it is more relaxed than in the period (E.g. around 2014) when there were affiliation changes that made immediate resignations necessary.  I'd agree with @frivoal and @chaals that I don't think this needs to be relaxed further at this time.

I DO continue to have a concern that Board affiliation changes do not prompt a re-evaluation by membership within a year, like they do on the TAG and AB.  It seems like we should be choosing one approach or the other.

**Comment by mnot** [comment]
> I DO continue to have a concern that Board affiliation changes do not prompt a re-evaluation by membership within a year, like they do on the TAG and AB. It seems like we should be choosing one approach or the other.

What is the driver for consistency here?

If there is a problematic change of affiliation, twenty members can force the issue. For that matter, they don't need such a trigger; they can call a special meeting at any time.

**Comment by cwilso** [comment]
I'll ask the question the other way around, since as I noted above, "it seems like we should be choosing one approach or the other."  

What is the driver to have the rule for the other bodies?  If I quit Google today, I would lose my seat automatically at the end of June 2024 (despite my term being for another year); why is that an important protection, while a Director changing affiliation has no effect?

(My personal assumption is that because despite all our talk that "TAG/AB/Directors rise above their affiliation", affiliation changes matter.)

**Comment by frivoal** [comment]
Addressed by merging https://github.com/w3c/process/pull/1167
